File: middlerAPP/src/app/components/identity/models/m-user-dto.ts

export class MUserDto {
    Id: string;
    UserName: string;
    Email: string;
    Firstname: string;
    Lastname: string;
    PhoneNumber: string;

    EmailConfirmed: boolean;
    PhoneNumberConfirmed: boolean;
    TwoFactorEnabled: boolean;
    LockoutEnabled: boolean;

    DateTime: Date | null;

    Active: boolean;
   
    Claims: Array<any>;
    Logins: Array<any>;
    Secrets: Array<any>;

    
}

File: middlerApp.API/IDP/DtoModels/MUserDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using middlerApp.API.IDP.Models;
using Org.BouncyCastle.Asn1;

namespace middlerApp.API.IDP.DtoModels
{
    public class MUserDto
    {

        public Guid Id { get; set; }

        [MaxLength(200)]
        public string UserName { get; set; }

        [MaxLength(200)]
        public string Email { get; set; }

        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }

        public bool EmailConfirmed { get; set; }
        public bool PhoneNumberConfirmed { get; set; }
        public bool TwoFactorEnabled { get; set; }
        public bool LockoutEnabled { get; set; }

        public DateTime? ExpiresOn { get; set; }

        public bool HasPassword { get; set; }

        public bool Active { get; set; }

        public ICollection<MUserClaimDto> Claims { get; set; } = new List<MUserClaimDto>();
        public ICollection<MUserLogin> Logins { get; set; } = new List<MUserLogin>();
        public ICollection<MUserSecret> Secrets { get; set; } = new List<MUserSecret>();

        public ICollection<MRoleListDto> Roles { get; set; } = new List<MRoleListDto>();

    }
}
